lib/inspec/resources/mssql_session.rb

```diff
@@ -42,11 +42,7 @@ def initialize(opts = {})
       @local_mode = opts[:local_mode]
       unless local_mode?
         @host = opts[:host] || "localhost"
-        if opts.key?(:port)
-          @port = opts[:port]
-        else
-          @port = "1433"
-        end
+        @port = opts[:port]
```

Comment:
What happens if someone neither specifies a port nor uses a named connection? In that case, shouldn't we default to a port?

Reply:
In that case also it works. It does not crash. MSSQL seems to establish connection even in that case. In my opinion it would work even without default port..but do you suggest we have the default port in case named instance is not used?

Reply:
OK, if you've tested it without a port, it seems fine to me.